cli/command/cli.go

```diff
@@ -34,6 +36,11 @@ import (
 	"github.com/theupdateframework/notary/passphrase"
 )
 
+const (
+	// ContextDockerHost is the reported context when DOCKER_HOST is used
+	ContextDockerHost = "<DOCKER_HOST>"
```

Comment:
The moby/moby/client stuff deals with `<DOCKER_HOST>` in some ways too. The separations of concerns is a bit unclear here.

Reply:
It should now be clearer. ContextDockerHost is now a special value used when resolving docker and kubernetes endpoints